webgoat-integration-tests/src/test/java/org/owasp/webgoat/IntegrationTest.java

```diff
@@ -21,12 +22,11 @@
 
 public abstract class IntegrationTest {
 
-    protected static int WG_PORT = 8080;
+    protected static int WG_PORT = 8081;
```

Comment:
Will this now always start WG when running the test locally? even if you have it running on port 8080 already it will start it again on port 8081?

Reply:
My mistake I will change it back to 8080. By the way the build is also succesful on Windows

Reply:
ok